community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CreateTest.scala

```diff
@@ -166,11 +160,11 @@ In this case we add a +Person+ label to the node as well.
 By providing Cypher an array of maps, it will create a node for each map.
 
 NOTE: When you do this, you can't create anything else in the same +CREATE+ clause.
-""",
+             """,
```

Comment:
I don't think will change the output of the doc, but it looks odd. Not very important anyhow.